addons/edi/models/edi_document.py

```diff
@@ -418,7 +420,11 @@ def commit_execute_progress(self, log_message):
             edi_progress = EDIDocProgress.search([("doc_id", "=", temp_self.id)])
             if not edi_progress:
                 edi_progress = EDIDocProgress.create({"doc_id": temp_self.id})
-            updated_log = "\n".join([edi_progress.progress_log or "", log_message])
+            # Update with the message, or reset it
+            if log_message:
```

Comment:
nit: 
```python
updated_log = ""
if log_message:
    updated_log = "\n".join([edi_progress.progress_log or "", log_message]).lstrip()
```

Reply:
The variable is set in the `else` part too (to reset the logs), so it will be defined in either case. But I'll set it to be explicit.

Reply:
We can drop the
```python
else:
    updated_log = ""
```

now